docs/geedocs/answer/3499938.html

```diff
@@ -62,18 +66,17 @@
 <li>SUSE Linux Enterprise Server 10</li>
```

Comment:
I think we should disclose that in the  landing page as one more OS and we can specify its not tests in OSS branch , or at least have a ticket to test it.